### ISSUE TITLE
PHP8: Warning: strtotime(): Epoch doesn't fit in a PHP integer in C:\inetpub\wwwroot\wire\core\Page.php on line 908

### ISSUE THREAD

**Issue by matjazpotocnik** [processwire/processwire-issues#1288]
PHP 8.0, latest PW dev:

Calling `strtotime($value)` when value is "0000-00-00 00:00:00" shows `Warning: strtotime(): Epoch doesn't fit in a PHP integer in C:\inetpub\wwwroot\wire\core\Page.php on line 908`

The same issue is in PageFinder.php:2531 and InputfieldDatetime.module:350

**Comment by ryancramerdesign** [comment]
@matjazpotocnik Thanks, I'll have a closer look here. Any idea where the `0000-00-00 00:00:00` date strings are coming from?

**Comment by matjazpotocnik** [comment]
@ryancramerdesign it's a long time (for my old head), but if I remember correctly, I looked at the database for pages and there it was. I think it was for creation/modification date/time for pages in the admin area. I don't know how these dates ended there initially, but maybe you could test for this scenario and return 'n/a' ?

**Comment by ryancramerdesign** [comment]
@matjazpotocnik I've pushed an attempted fix for this. My MySQL won't accept dates with all zeros, and despite repeated attempts, I can't get PHP8 to successfully install, so don't have a good way to test this one. Please let me know if you find the committed fix doesn't resolve it. Thanks.

**Comment by matjazpotocnik** [comment]
@ryancramerdesign I can confirm the fix is working for me. I manually inserted "0000-00-00 00:00:00" in created|published|modified fields in the database for one specific page and then edited a page, no warning.

While peeking at the database, I noticed that some pages have created_users_id value of 2. However, I can't find a user with the id of 2. Is this ok? It could be that my PW installation got somehow corrupted, and thats' how I got those dates in the DB? Whatever the reason, it's fixed, thank you for your time.
